optuna/visualization/matplotlib/_param_importances.py

```diff
@@ -90,37 +94,66 @@ def objective(trial):
 
     _imports.check()
 
-    importances_info = _get_importances_info(study, evaluator, params, target, target_name)
-    return _get_importances_plot(importances_info)
+    if target or not study._is_multi_objective():
+        importances_infos: tuple[_ImportancesInfo, ...] = (
+            _get_importances_info(study, evaluator, params, target, target_name),
+        )
 
+    else:
+        n_objectives = len(study.directions)
+        importances_infos = tuple(
+            _get_importances_info(
+                study,
+                evaluator,
+                params,
+                target=lambda t: t.values[objective_id],
+                target_name=f"{target_name} {objective_id}",
+            )
+            for objective_id in range(n_objectives)
+        )
 
-def _get_importances_plot(info: _ImportancesInfo) -> "Axes":
+    return _get_importances_plot(importances_infos)
+
+
+def _get_importances_plot(infos: tuple[_ImportancesInfo, ...]) -> "Axes":
     # Set up the graph style.
     plt.style.use("ggplot")  # Use ggplot style sheet for similar outputs to plotly.
     fig, ax = plt.subplots()
-    ax.set_title("Hyperparameter Importances")
-    ax.set_xlabel(f"Importance for {info.target_name}")
+    ax.set_title("Hyperparameter Importances", loc="left")
+    ax.set_xlabel("Hyperparameter Importance")
     ax.set_ylabel("Hyperparameter")
+    height = 0.8 / len(infos)  # Default height split between objectives.
+
+    for objective_id, info in enumerate(infos):
+        param_names = info.param_names
+        pos = np.arange(len(param_names))
+        offset = height * objective_id
+        importance_values = info.importance_values
+
+        if not importance_values:
+            continue
+
+        # Draw horizontal bars.
+        ax.barh(
+            pos + offset,
+            importance_values,
+            height=height,
+            align="center",
+            label=info.target_name,
+            color=plt.get_cmap("tab20c")(objective_id),
```

Comment:
BTW, this is not critical at all, but the number of objectives is limited to 20 here. It might also be a reasonable follow-up task to allow users to pass their own color list, defaulting to plt.get_cmap("tab20c").

Reply:
Sounds good.

Reply:
For now, we can mod `objective_id` by the number of cmap.